src/features/pages/deriv-ez/hero/index.tsx

```diff
@@ -18,37 +22,52 @@ const DerivEZHero = () => {
             align={'center'}
             md={{ direction: 'row' }}
         >
-            <FlexBox.Box
-                className={hero_wrapper}
-                direction={'col'}
-                justify={'center'}
-                ml={'8x'}
-                margin_block={'15x'}
-                md={{ ml: '0x' }}
-            >
-                <FlexBox.Item className={hero_image} align_self="start">
-                    <div className="visible-phone-and-tablet">{hero_images['mobile_logo']}</div>
-                    <div className="visible-larger-than-tablet">{hero_images['logo']}</div>
-                </FlexBox.Item>
-                <Typography.Heading
-                    size={'xlarge'}
-                    color={'secondary'}
-                    weight={'bold'}
-                    align={'left'}
-                    pt={'4x'}
-                    md={{ pt: '8x' }}
+            <FlexBox.Box container={'fluid'} justify={'center'} md={{ justify: 'start' }}>
+                <FlexBox.Box
+                    className={hero_wrapper}
+                    direction={'col'}
+                    ml={'8x'}
+                    margin_block={'15x'}
+                    md={{ ml: '0x' }}
                 >
-                    <Localize translate_text="_t_An intuitive,_t_" />
-                    <br />
-                    <Localize translate_text="_t_easy-to-get-started_t_" />
-                    <br />
-                    <Localize translate_text="_t_CFDs trading platform_t_" />
-                </Typography.Heading>
+                    <FlexBox.Item className={hero_image} align_self="start">
+                        <div className="visible-phone-and-tablet">{hero_images['mobile_logo']}</div>
+                        <div className="visible-larger-than-tablet">{hero_images['logo']}</div>
+                    </FlexBox.Item>
+                    <Typography.Heading
+                        size={'xlarge'}
+                        color={'secondary'}
+                        weight={'bold'}
+                        align={'left'}
+                        pt={'4x'}
+                        mb={'12x'}
+                        md={{ pt: '8x', mb: '20x' }}
+                    >
+                        <Localize translate_text="_t_An intuitive CFD trading platform for desktop and mobile_t_" />
```

Comment:
can we put inside a data file ?

Reply:
it'll done when we implement correct template. since we have design differenties we can keep it